README.md

```diff
@@ -1,2 +1,87 @@
 # isar-turtlebot
-ISAR implementation for the Turtlebot3 Waffle Pi
+ISAR implementation for the Turtlebot3 Waffle Pi.
+
+# Installation for development
+
+For local development, please fork the repository. Then, clone and install in the repository root folder:
+
+```bash
+$ git clone https://github.com/equinor/isar-turtlebot
+$ cd isar-turtlebot
+$ pip install -r requirements.txt -r requirements-dev.txt
+$ pip install -e .
+```
+# Simulation prerequisities
+
+The simulator requires a computer or a virtual machine running Ubuntu 20. To install and run the turtlebot simulator, you will need to [install ROS noetic](http://wiki.ros.org/noetic/Installation/Ubuntu) on your computer. The `desktop` version should be sufficient.
+
+
+Then, [install dependent ROS packages](https://emanual.robotis.com/docs/en/platform/turtlebot3/quick-start/), and the [simulation package](https://emanual.robotis.com/docs/en/platform/turtlebot3/simulation/).
+
+You also need to install gazebo:
+```bash
+$ sudo apt-get install gazebo11
+```
+The installation can be verified by running:
+```bash
+$ gazebo
+```
+
+Install the required [ROS packages](
+https://gazebosim.org/tutorials?tut=ros_installing&cat=connect_ros) for gazebo:
+
+```bash
+$ sudo apt-get install ros-noetic-gazebo-ros-pkgs ros-noetic-gazebo-ros-control
+```
+
+And rosbridge:
+```bash
+$ sudo apt-get install ros-noetic-rosbridge-suite
+```
+
+
+# Running the simulation
+
+First, you will need to set the following environment variable:
+
+```bash
```

Comment:
Add a description of how to add this to the .bashrc file, since this is needed across several terminals and is unlikely to be updated?